src/SprykerEco/Zed/Payone/Business/Payment/DataMapper/OrderHandlingMapper.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Payone\Business\Payment\DataMapper;
+
+use Generated\Shared\Transfer\OrderTransfer;
+use Spryker\Shared\Shipment\ShipmentConfig;
+use SprykerEco\Shared\Payone\PayoneApiConstants;
+use SprykerEco\Zed\Payone\Business\Api\Request\Container\AbstractRequestContainer;
+
+class OrderHandlingMapper implements OrderHandlingMapperInterface
```

Comment:
* ExpenseMapper.

Reply:
It maps expenses except shipment. 
Naming ExpenseMapper tells that it maps all expenses

Reply:
++